src/main/resources/static/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Produkty</title>
+</head>
+<body>
+<a href="/lista">wyświetl wszystkie produkty</a><br/>
+<a href="/lista?kategoria=FOOD">wyświetl produkty z kategorii art. spożywcze</a><br/>
+<a href="/lista?kategoria=HOUSEHOLD">wyświetl produkty z kategorii art. gospodarstwa domowego</a><br/>
+<a href="/lista?kategoria=OTHER">wyświetl produkty z kategorii inne</a>
```

Comment:
Używaj atrybutu th:href (pokazane w review), najlepiej gdyby ta lista była wygenerowana na podstawie enuma, podobnie jak omawiam to w kontekście formularza, czyli jeżeli pojawi się nowa kategoria, albo jakąś usuniemy, to zmiana ta powinna być automatycznie uwzględniona